src/ConfigManager.cpp

```diff
@@ -192,26 +212,28 @@ void ConfigManager::setWifiConnectInterval(const int interval) {
 }
 
 bool ConfigManager::wifiConnected() {
+  return WiFi.status() == WL_CONNECTED;
```

Comment:
added this as a public method so something else can see if the wifi is still connected and kick off and reboot/etc ...